### PR TITLE
Make Sling-Initial-Content more specific

### PR DESCRIPTION
**Pull request by kwin**
This prevents overwriting other nodes and ACLs at /apps, /content or /etc

### CODE REVIEW

**Review by raducotescu — APPROVED** [review]

File: bnd.bnd [hunk 1]
```diff
@@ -1,2 +1,2 @@
-Sling-Initial-Content: SLING-INF;overwrite=true
-Require-Capability:    io.sightly; filter:="(&(version>=1.0)(!(version>=2.0)))"
\ No newline at end of file
+Sling-Initial-Content: SLING-INF/apps/repl/components/repl;overwrite=true;path=/apps/repl/components/repl,SLING-INF/content/htl/repl.json;overwrite=true;path=/content/htl/repl,SLING-INF/etc/clientlibs/repl;overwrite=true;path=/etc/clientlibs/repl
```
Comment (stefanseifert) [inline bnd.bnd]: cosmetic remark: would be better readable in multiple lines

**Review by stefanseifert — APPROVED** [review]

**Review by rombert — CHANGES_REQUESTED** [review]
I think something is off here. I have built the PR locally ( from `org.apache.sling.scripting.sightly.repl-1.0.8-2-g171c66` )

and rebuilt the starter locally

```diff
diff --git a/src/main/features/app/htl_repl.json b/src/main/features/app/htl_repl.json
index 9d7ed91..dc87837 100644
--- a/src/main/features/app/htl_repl.json
+++ b/src/main/features/app/htl_repl.json
@@ -1,8 +1,8 @@
 {
     "bundles": [
         {
-            "id":"org.apache.sling:org.apache.sling.scripting.sightly.repl:1.0.6",
+            "id":"org.apache.sling:org.apache.sling.scripting.sightly.repl:1.0.9-SNAPSHOT",
             "start-order":"20"
         }
     ]
-}
\ No newline at end of file
+}
```

and then launching the Composum Browser I see the nodes plastered all over the root directory

image

Does this happen to you as well?

**Comment by rombert** [comment]
(a jira reference would be nice as well :-) )

**Comment by rombert** [comment]
I think 2346d5c  fixes it.

**Review by stefanseifert — APPROVED** [review]

**Comment by kwin** [comment]
> (a jira reference would be nice as well :-) )

Done in SLING-10987

> I think something is off here...

@rombert Good catch, path must be directives i.e. use separator `:=`

**Pull request merged**